### PR TITLE
Update Falskaar

### PR DESCRIPTION
**Pull request by ddbb07**
- Added patch messages
- Added useInstead messages for outdated patches

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -12469,6 +12469,34 @@ plugins:
   - name: 'Falskaar.esm'
     url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/2057/' ]
     msg:
+      - <<: *alsoUseX
+        subs: [ '[Falskaar - Addons and Patches](https://www.nexusmods.com/skyrimspecialedition/mods/19454/)' ]
```
Comment (sibir-ine) [inline masterlist.yaml]:
```suggestion
        subs: [ '[Falskaar - Bug Fixes](https://www.nexusmods.com/skyrimspecialedition/mods/19454/)' ]
```
Just to make it clear what plugin is being recommended.

File: masterlist.yaml [hunk 2]
```diff
@@ -12483,6 +12511,17 @@ plugins:
         crc: 0x00AFAC80
         util: '[SSEEdit v4.0.3](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
         itm: 3
+    
+  - name: 'Falskaar - Bruma Patch.esp'
+    msg:
+      - <<: *useInstead
+        subs: [ '[Legacy of the Dragonborn Patches (Official)](https://www.nexusmods.com/skyrimspecialedition/mods/30980/)' ]
```
Comment (sibir-ine) [inline masterlist.yaml]: I'm not sure what's going on here. Why should users of an outdated Bruma & Falskaar patch use a patch (or patches) for LotD instead?

File: masterlist.yaml [hunk 3]
```diff
@@ -12483,6 +12511,17 @@ plugins:
...
+    
+  - name: 'Falskaar - Unique Region Names.esp'
+    msg:
+      - <<: *useInstead
+        subs: [ '[Unique Region Names](www.nexusmods.com/skyrimspecialedition/mods/11750/)' ]
```
Comment (sibir-ine) [inline masterlist.yaml]:
```suggestion
        subs: [ '[Unique Region Names - Falskaar addon](www.nexusmods.com/skyrimspecialedition/mods/11750/)' ]
```
Again, just to make it clear what's being recommended.

**Review by sibir-ine — CHANGES_REQUESTED** [review]
@ddbb07 Besides my other comments, the rest looks good.

While your at it, would you mind moving the entries from Falskaar - Addons and Patches, so:
```yaml
  ### Falskaar - Addons and Patches ###
  - name: 'Falskaar - Bruma Patch.esp'
    ...
  - name: 'Falskaar - Bug Fixes.esp'
    ...
  - name: 'Falskaar - Unique Region Names.esp'
    ...
```
to the 3RD Party Patches section in alphabetical order?

Also, could you remove the cleaning info from the `Falskaar - Bug Fixes.esp` entry, so:
```yaml
    clean:
      # version: 1.6
      - crc: 0xC7B6E6E8
        util: 'SSEEdit v4.0.2f'
```
since it's a patch & not dirty? No worries if not; I can take care of that later.

**Comment by ddbb07** [comment]
Sorry for this taking so long, but im done now.

File: masterlist.yaml [hunk 4]
```diff
@@ -12469,6 +12469,44 @@ plugins:
...
+        subs: [ '[Falskaar - Bug Fixes](https://www.nexusmods.com/skyrimspecialedition/mods/19454/)' ]
+        condition: 'not active("Falskaar - Bug Fixes.esp")'
+      - <<: *patch3rdParty
+        subs:
+          - 'Better Dynamic Snow'
+          - '[Falskaar - Addons and Patches](https://www.nexusmods.com/skyrimspecialedition/mods/19454/)'
+        condition: 'active("Better Dynamic Snow.esp") and not active("Falskaar - Better Dynamic Snow Patch.esp")'
+      - <<: *patch3rdParty
+        subs:
+          - Beyond Skyrim - Bruma'
```
Comment (sibir-ine) [inline masterlist.yaml]: @ddbb07 Besides this, it looks good to go.
```suggestion
          - 'Beyond Skyrim - Bruma'
```

Comment (ddbb07) [reply masterlist.yaml]: Oh, how did i miss that, thanks for the suggestion

**Comment by sibir-ine** [comment]
Thanks!

**Pull request merged**